website/content/en/pre-docs/deploy-guide/_index.md

```diff
@@ -0,0 +1,165 @@
+---
+title: "Deployment Guide"
+linkTitle: "Deployment Guide"
+weight: 50
+---
+
+Introducing new software the modifies how Kubernetes operates is complex, and no single solution exists. 
+
+This guide describes how Karpenter interacts with external resources, and modifies cluster behavior. 
+
+## Resources Dependencies
+
+### Cluster on AWS
+
+Currently, Karpenter creates nodes (EC2 Instances) through the AWS cloud provider module, the only such module implemented. Clusters on other cloud providers are contemplated, but are not currently supported. 

```

Comment:
```suggestion
Karpenter creates nodes (EC2 Instances) through the AWS cloud provider, the only such module implemented.
Node provisioning on other providers is not currently supported.
If you would like to use Karpenter with other infrastructure providers please open an issue to let us know your use case.
```
Do we call it a "module" other places. I think for users they don't need to know the terminology in the code